homeassistant/components/tuya/sensor.py

```diff
@@ -476,6 +476,32 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        TuyaSensorEntityDescription(
+            key=DPCode.WORK_STATE,
+            name="Mode",
+            device_class=TuyaDeviceClass.WORK_STATE,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        # Total seconds of irrigation. Read-write value; the device appears to ignore the write action (maybe firmware bug)
+        TuyaSensorEntityDescription(
+            key=DPCode.TIME_USE,
+            name="Total Watering Time",
+            icon="mdi:history",
+            state_class=SensorStateClass.TOTAL_INCREASING,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        # Number of seconds for the watering session. The device will automatically turn off after this time
+        TuyaSensorEntityDescription(
+            key=DPCode.COUNTDOWN,
+            name="Countdown",
+            icon="mdi:timer-outline",
```

Comment:
Does this count down seconds during watering? like 10... 9... 8...?

Reply:
The count down is the number of seconds until it turns off (Closed the watter)